### PR TITLE
Add Blog, Add Student

### PR DESCRIPTION
**Pull request by amryarios**
Please read and understand everything below
**Do not delete any text other than where you are instructed.**

**Students: If one of them is applicable to you. Please check it.**

Check by changing each `[ ]` to `[x]` Please take note of the whitespace as it matters.

- [ ] Read and understood (see CONTRIBUTING.md)
- [ ] Included a Preview link and screenshot showning after and before the changes.
- [ ] Images are `240 x 240` [w x h].
- [ ] Included a description of change below.
- [ ] Squashed the commits.

# Changes done in this Pull Request

- If your change will be reflected on the website, please provide a **Test-Link** (**Hint : `gh-pages`**)
- Fixes`#<Issue no.>`

### CODE REVIEW

File: _data/blogs.yml [hunk 1]
```diff
@@ -1,3 +1,8 @@
+- url: https://demiprestasi.wordpress.com/2017/12/23/interview-with-a-teacher-about-fossasia-and-google-code-in/
+  title: Interview with a teacher about FOSSASIA and Google Code-in
+  img: me1.jpg
```
Comment (abishekvashok) [inline _data/blogs.yml]: rename the image to something meaningful please and this is ready to go

**Review by abishekvashok — CHANGES_REQUESTED** [review]

**Review by nikit19 — CHANGES_REQUESTED** [review]
Please resolve conflicts

File: _data/blogs.yml (hunk 1 above)
Comment (amryarios) [reply _data/blogs.yml]: Why ? because the image for my blog named me ?
is it not allowed?

**Comment by abishekvashok** [comment]
@nikit19 I guess they were resolved.

**Comment by amryarios** [comment]
I have already done @nikit19

**Comment by ParthS007** [comment]
@amryarios  Ok no problem, If you are unable to change image name but why you are adding two images.Please remove `amryarios.jpg` and else looks fine 👍

**Comment by amryarios** [comment]
@ParthS007 thank you, for the picture of amryarios.jpg it is to enter student data

**Comment by amryarios** [comment]
Check https://amryarios.github.io/gci17.fossasia.org/

**Comment by ParthS007** [comment]
@amryarios  Can you please update your preview link , I am not able to see the changes in blog posts. Thank you 👍

File: _data/blogs.yml (hunk 1 above)
Comment (abishekvashok) [reply _data/blogs.yml]: Oh ok.

**Comment by amryarios** [comment]
You can check here https://amryarios.github.io/gci17.fossasia.org/
see the changes on : 
- Blog
- Students just like you.
Thank You :+1:  @ParthS007

**Comment by ParthS007** [comment]
image
@amryarios  see I am not able to see the changes , can you share the screenshot ?

**Comment by amryarios** [comment]
untitled
untitled1
@ParthS007

**Comment by ParthS007** [comment]
@amryarios  Why blog images are not loading, can you please see, Student entry is fine 👍

**Comment by amryarios** [comment]
yeah, i am still confused why the blog image does not appear when i do not make anything just add pictures and data only @ParthS007

**Comment by ParthS007** [comment]
@abishekvashok  Can you help @amryarios  ?

**Comment by nikit19** [comment]
@amryarios please add the blog image in the img/blog folder

**Comment by amryarios** [comment]
Sorryyy :( , wrongly add a picture on the blog, I added it in the student section but now I have done @nikit19

**Comment by ParthS007** [comment]
@amryarios  Update preview link

**Comment by amryarios** [comment]
https://amryarios.github.io/gci17.fossasia.org/ @ParthS007

**Comment by amryarios** [comment]
image
in my already appeared whether in you have appeared? @ParthS007

**Review by ParthS007 — APPROVED** [review]

**Review by harshit98 — APPROVED** [review]

**Pull request merged**